src/test/java/pages/SearchPage.java

```diff
@@ -0,0 +1,128 @@
+package pages;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class SearchPage extends BasePage{
+
+    public SearchPage(WebDriver browser){
+        super(browser);
+    }
+
+    public String getSearchResultKeyword() {
+         WebElement searchResult = browser.findElement(By.name("displayterm"));
+         wait.until(ExpectedConditions.visibilityOf(searchResult));
+         return searchResult.getAttribute("value");
+    }
+
+    public List<WebElement> getSearchItems(){
```

Comment:
This method cannot be public, because all work with elements must be done inside the page class